microprofile/reactive-messaging-smallrye/kafka/src/main/java/org/wildfly/microprofile/reactive/messaging/config/kafka/ssl/context/WildFlyKafkaSSLEngineFactory.java

```diff
@@ -20,18 +17,22 @@
 import javax.net.ssl.SSLEngine;
 import javax.net.ssl.SSLParameters;
 
+import org.wildfly.microprofile.reactive.messaging.common.security.ElytronSSLContextRegistry;
+
 /**
  * @author <a href="mailto:kabir.khan@jboss.com">Kabir Khan</a>
  */
 public class WildFlyKafkaSSLEngineFactory implements org.apache.kafka.common.security.auth.SslEngineFactory {
+    static final String SSL_ENGINE_FACTORY_CLASS = "ssl.engine.factory.class";
```

Comment:
Nit for future consideration: This constant is declared here but isn't used here. AFAICT it's only used in a different class.